scipy/spatial/transform/tests/test_rotation.py

```diff
@@ -815,35 +771,26 @@ def test_as_euler_degenerate_compare_algorithms():
 
     idx = angles[:, 1] == 0  # find problematic angles indices
 
+    # Extrinsic rotations
+    seq = "".join([seq_tuple[0], seq_tuple[1], seq_tuple[0]])
+    if intrinsic:
+        seq = seq.upper()
+    rot = Rotation.from_euler(seq, angles, degrees=True)
```

Comment:
I don't mind pulling out the processing, but the comments could be clearer IMO.

```suggestion
    # Extrinsic rotations
    seq = "".join([seq_tuple[0], seq_tuple[1], seq_tuple[0]])
    if intrinsic:
        # Intrinsic rotations
        seq = seq.upper()

    rot = Rotation.from_euler(seq, angles, degrees=True)
```